core/src/main/resources/lib/layout/copyButton/copyButton.js

```diff
@@ -11,6 +11,10 @@ Behaviour.specify("span.copy-button", 'copyButton', 0, function(e) {
         el.style.height = "1px";
         el.style.border = "none";
         el.style.padding = "0px";
+        el.style.position = "absolute";
+        el.style.top = "-99999px";
+        el.style.left = "-99999px";
+        el.style.tabindex = "-1";
```

Comment:
tabindex is not style, is an attribute and should be set with `setAttribute` as a string https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute